src/library.js

```diff
@@ -3226,11 +3226,13 @@ LibraryManager.library = {
     {{{ cDefine('ESTRPIPE') }}}: 'Streams pipe error',
   },
   __setErrNo: function(value) {
+#if SUPPORT_ERRNO
     if (Module['___errno_location']) {{{ makeSetValue("Module['___errno_location']()", 0, 'value', 'i32') }}};
 #if ASSERTIONS
     else err('failed to set errno from JS');
 #endif
     return value;
+#endif
```

Comment:
This function returns the value, so I think it should return 0 here. That shouldn't be a problem for closure, in fact I'd guess it might be better (no need to think about `undefined`), but worth checking.